src/stdlib_random.fypp

```diff
@@ -18,7 +18,7 @@ module stdlib_stats_distribution_PRNG
     !! Version experimental
     !!
     !! Generation of random integers with different kinds
-    !! ([Specification](../page/specs/stdlib_stats_distribution_PRNG.html#
+    !! ([Specification](../page/specs/stdlib_random.html#
     !! description))
```

Comment:
While modiffyng this module, is it also ok to update the link to a more explicit one (instead of desctiption)?

Reply:
Done.